teachers.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# subclass for the person
+class Teacher < Person
+  def initialize(id, age, specialization, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] Please kindly structure this as well with the example codes i have provided to you :+1: